content/en/workflows/access.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Access and Authentication
+kind: documentation
+description: Access and authentication for Workflows
+aliases:
+  - "/workflows/service_accounts"
+  - "/workflows/setup"
```

Comment:
```suggestion
  - "/workflows/setup"
```
I wonder if we should move this alias to the connections page seeing as though the content the user is most likely to have bookmarked is on that page? I could see it both ways though.

Reply:
I see what you mean. The majority of the information that was on the "setup" page is now on "connections." I'll move the alias.